pyccl/neutrinos.py

```diff
@@ -2,6 +2,13 @@
 from pyccl import ccllib as lib
 from pyccl.pyutils import check
 
+neutrino_mass_splits = {
```

Comment:
I like this name for these settings -- could maybe use it in the C code as well?

Reply:
Good idea, change made.